hubmap/query_app/set_evaluators.py

```diff
@@ -59,6 +60,17 @@
 )
 
 
+def copy_pagination_format(results_json: str) -> str:
+    response_dict = {}
+    response_dict["count"] = 1
+    response_dict["next"] = None
+    response_dict["previous"] = None
+    response_dict["results"] = []
+    response_json = json.dumps(response_dict)
+    response_json = response_json.replace(json.dumps([]), results_json)
```

Comment:
This looks really strange -- can you add a comment explaining why this is necessary? And isn't `json.dumps([])` just `'[]'`?